LiveIngest/LiveEventWithDVR/src/main/java/sample/LiveEventWithDVR.java

```diff
@@ -89,17 +76,17 @@ private static void runLiveEvent(ConfigWrapper config) {
             // Note: When creating a LiveEvent, you can specify allowed IP addresses in one of the following formats:                 
             //       IpV4 address with 4 numbers
             //       CIDR address range
-            IPRange allAllowIPRange = new IPRange().withName("AllowAll").withAddress("0.0.0.0").withSubnetPrefixLength(0);
-            List<IPRange> listIPRanges = new ArrayList<>();
+            IpRange allAllowIPRange = new IpRange().withName("AllowAll").withAddress("0.0.0.0").withSubnetPrefixLength(0);
```

Comment:
Might look nicer to move each with to it's own line here.

Reply:
Move the .with fluid statements to their own lines consistently.